README.md

```diff
@@ -4,4 +4,48 @@ bitcoin-seed
 [![NPM Package](https://img.shields.io/npm/v/bitcoin-seed.svg?style=flat-square)](https://www.npmjs.org/package/bitcoin-seed)
 [![Build Status](https://img.shields.io/travis/ExodusMovement/bitcoin-seed.svg?branch=master&style=flat-square)](https://travis-ci.org/ExodusMovement/bitcoin-seed)
 
-Don't use yet.
+## API
+
+### `fromRandom({ passphrase, entropyFn })`
+
+Creates a random new `seed` instance with optional `passphrase`. If `entropyFn` is passed, it must return a 16-byte random `Buffer`; `crypto.randomBytes(16)` is used by default.
```

Comment:
Not sure if `entropyFn` should be documented, or if it's just there for tests.

Reply:
it looks like a useful param to me, to keep it platform-agnostic. We should never have any public API params just for tests imo